pkg-new/license/signature_test.go

```diff
@@ -0,0 +1,401 @@
+package license
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/helpers"
+	kotsv1beta1 "github.com/replicatedhq/kotskinds/apis/kots/v1beta1"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//go:embed testdata/*
+var testdata embed.FS
+
+func Test_VerifySignature(t *testing.T) {
+	req := require.New(t)
+
+	// Read license with modified licenseID field
+	licenseBytes, err := testdata.ReadFile("testdata/invalid-changed-licenseID.yaml")
+	req.NoError(err)
+
+	license, err := helpers.ParseLicenseFromBytes(licenseBytes)
+	req.NoError(err)
+
+	// Verify signature should detect the tampered licenseID field
```

Comment:
what about verifying an invalid signature as well, since that's the main purpose of the test?